src/vendor/requests.js

```diff
@@ -0,0 +1,103 @@
+/* global fetch */
+import { parse } from 'query-string';
+import { leaves, toPairs } from './vectors';
+import { Log } from './logs';
+import { Data } from './Data';
+import { exists, isArray, isString, toArray } from './utils';
+import strings from './strings';
+
+function fromQueryString(query) {
```

Comment:
At a first peak, I'm uncertain about the format of the query it expects. Does it handle both `?search=..` and `search=`? A comment above this function would be helpful :)

Are there any tests configured for this utility by any chance?

Reply:
I will add description, and move the tests (which are [here](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/9684d44e481a20e624e0c41e53caaf1162261e99/test/vendor/convert.test.js#L75))